sw/device/lib/dif/dif_otp_ctrl.c

```diff
@@ -229,34 +235,37 @@ dif_result_t dif_otp_ctrl_get_status(const dif_otp_ctrl_t *otp,
   }
 
   static const bitfield_bit32_index_t kIndices[] = {
-      [kDifOtpCtrlStatusCodeVendorTestError] =
-          OTP_CTRL_STATUS_VENDOR_TEST_ERROR_BIT,
-      [kDifOtpCtrlStatusCodeCreatorSwCfgError] =
-          OTP_CTRL_STATUS_CREATOR_SW_CFG_ERROR_BIT,
-      [kDifOtpCtrlStatusCodeOwnerSwCfgError] =
-          OTP_CTRL_STATUS_OWNER_SW_CFG_ERROR_BIT,
-      [kDifOtpCtrlStatusCodeRotCreatorAuthCodesignError] =
-          OTP_CTRL_STATUS_ROT_CREATOR_AUTH_CODESIGN_ERROR_BIT,
-      [kDifOtpCtrlStatusCodeRotCreatorAuthStateError] =
-          OTP_CTRL_STATUS_ROT_CREATOR_AUTH_STATE_ERROR_BIT,
-      [kDifOtpCtrlStatusCodeHwCfg0Error] = OTP_CTRL_STATUS_HW_CFG0_ERROR_BIT,
-      [kDifOtpCtrlStatusCodeHwCfg1Error] = OTP_CTRL_STATUS_HW_CFG1_ERROR_BIT,
-      [kDifOtpCtrlStatusCodeSecret0Error] = OTP_CTRL_STATUS_SECRET0_ERROR_BIT,
-      [kDifOtpCtrlStatusCodeSecret1Error] = OTP_CTRL_STATUS_SECRET1_ERROR_BIT,
-      [kDifOtpCtrlStatusCodeSecret2Error] = OTP_CTRL_STATUS_SECRET2_ERROR_BIT,
-      [kDifOtpCtrlStatusCodeLifeCycleError] =
-          OTP_CTRL_STATUS_LIFE_CYCLE_ERROR_BIT,
-      [kDifOtpCtrlStatusCodeDaiError] = OTP_CTRL_STATUS_DAI_ERROR_BIT,
-      [kDifOtpCtrlStatusCodeLciError] = OTP_CTRL_STATUS_LCI_ERROR_BIT,
-      [kDifOtpCtrlStatusCodeTimeoutError] = OTP_CTRL_STATUS_TIMEOUT_ERROR_BIT,
-      [kDifOtpCtrlStatusCodeLfsrError] = OTP_CTRL_STATUS_LFSR_FSM_ERROR_BIT,
-      [kDifOtpCtrlStatusCodeScramblingError] =
-          OTP_CTRL_STATUS_SCRAMBLING_FSM_ERROR_BIT,
-      [kDifOtpCtrlStatusCodeKdfError] = OTP_CTRL_STATUS_KEY_DERIV_FSM_ERROR_BIT,
-      [kDifOtpCtrlStatusCodeBusIntegError] =
-          OTP_CTRL_STATUS_BUS_INTEG_ERROR_BIT,
-      [kDifOtpCtrlStatusCodeDaiIdle] = OTP_CTRL_STATUS_DAI_IDLE_BIT,
-      [kDifOtpCtrlStatusCodeCheckPending] = OTP_CTRL_STATUS_CHECK_PENDING_BIT,
+#ifdef OPENTITAN_IS_EARLGREY
+    [kDifOtpCtrlStatusCodeVendorTestError] =
```

Comment:
Is this correct? It looks like this also exists on darjeeling.

Reply:
I've included the partitions that DJ has